src/pipecat/processors/aggregators/openai_llm_context.py

```diff
@@ -112,11 +112,59 @@ def get_messages_for_logging(self) -> str:
             msgs.append(msg)
         return json.dumps(msgs)
 
-    def from_standard_message(self, message):
+    def from_standard_message(self, message) -> dict:
```

Comment:
I think this change isn't necessary.

The OpenAI format allows `content` to be a string or a list. The list elements can be of the form `{ "type": "text", "text": text }`.

If we do make a change like this, we have to handle non-text content parts properly. (Image, audio.)

Reply:
Reverted + added docstring documenting behavior.